p2p/peer_error.go

```diff
@@ -73,6 +73,7 @@ const (
 	DiscSelf
 	DiscReadTimeout
 	DiscSubprotocolError
+	DiscThrottled
```

Comment:
Ah, wanted to as about that. What error should I return in the case of throttling then?